inst/BS5/assets/pkgdown.scss

```diff
@@ -3,11 +3,16 @@
 .row > main {
   // Ensure contents never become unreadably wide
   max-width: 50rem;
-  
-  // And that we can break and hypenate very long words
-  overflow-wrap: break-word;
-  hyphens: auto;
 }
+
+@media (max-width: 800px) {
```

Comment:
This seems like a reasonable threshold, based on https://stackoverflow.com/questions/13847755/css-media-queries-for-screen-sizes

@dvg-p4 can you take a look at the PR preview?

Reply:
I think it would be better to match the existing bootstrap sizes. However, I don't remember what they are 😞

Reply:
Used the sass mixin instead.